examples/bearriver/src/FRP/BearRiver.hs

```diff
@@ -153,6 +155,21 @@ after q x = feedback q go
                   ct = if t' < 0 then constant (NoEvent, t') else go
               return ((e, t'), ct)
 
+occasionally :: MonadRandom m 
+            => Time -- ^ The time /q/ after which the event should be produced on average
+            -> b    -- ^ Value to produce at time of event
+            -> SF m a (Event b)
+occasionally tAvg b
+  | tAvg <= 0 = error "dunai: Non-positive average interval in occasionally."
+  | otherwise = proc _ -> do
+    r <- getRandomRS (0, 1) -< ()
```

Comment:
Indent one level into? (add  2 spaces to the arrow block).

Reply:
And align the `<-`

Reply:
Not sure what you mean with one level into 2 spaces to the arrow block: it is already on level 2 (has 4 spaces). Added another level (6 spaces now).